templater/templater_test.go

```diff
@@ -34,4 +34,33 @@ var _ = Describe("Templater", func() {
 			Expect(content).To(ContainSubstring(`Responsible Role: Amazon Elastic Compute Cloud: AWS Staff`))
 		})
 	})
+	Describe("DiffSSP", func() {
+		It("should warn the user if the current SSP contains a responsible role that conflicts with the " +
+			"responsbile role in the YAML", func() {
+
+			By("Loading the SSP with the Responsible Role being 'OpenControl Role Placeholder' " +
+				"for Control 'AC-2 (1)'")
```

Comment:
> While By() is useful, it feels like an antipattern...mind splitting some of this test out to helper functions?

I didn't know `By` existed either lol. Felt right to break it up with text. But i added some helpers too.